CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -1420,36 +1476,18 @@ _CFThreadSpecificKey _CFThreadSpecificKeyCreate() {
 
 CFTypeRef _Nullable _CFThreadSpecificGet(_CFThreadSpecificKey key) {
 #if TARGET_OS_WIN32
-    _CFThreadSpecificData *data = (_CFThreadSpecificData *)FlsGetValue(key);
-    if (data == NULL) {
-        return NULL;
-    }
-    return data->value;
+  return (CFTypeRef)FlsGetValue(key);
```

Comment:
```suggestion
    return (CFTypeRef)FlsGetValue(key);
```

Reply:
I've seen a few like this. Especially wrt the implementation of TSDs on Windows, @compnerd, let me know if it looks bad?

Reply:
My mental map of The Whole Of Foundation gets a little murky around the Windows bits.